common/accel-dashboard.cpp

```diff
@@ -0,0 +1,54 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#include "motion-dashboard.h"
+
+using namespace rs2;
+using namespace rsutils::string;
+
+accel_dashboard::accel_dashboard( std::string name )
+    : motion_dashboard( name )
+{
+}
+
+void accel_dashboard::process_frame( rs2::frame f )
```

Comment:
No need for two separate files. The difference between gyro and accel is the checked stream type. The object can get it as a parameter in the constructor. The shared 
code can be in `motion-dashboard.cpp`

Reply:
A new parameter is added to a constructor.